chart-parts/ingress.yaml

```diff
@@ -1,40 +1,39 @@
-{{- if .Values.services.ingress -}}
+{{- if .Values.ingress.enabled -}}
 ---
-# The cert and the key cannot be filled automatically. If we could it
-# would roughly look like the lines below. As is it is the
-# responsibility of the operator to retrieve the values of these
-# secrets from the deployed UAA and update this secret with the actual
-# values. And the responsibility of the developer to supply the proper
-# instructions to the operator.
-##
-# tls.crt: {{ default "" .Values.secrets.UAA_SERVER_CERT     | b64enc | quote }}
-# tls.key: {{ default "" .Values.secrets.UAA_SERVER_CERT_KEY | b64enc | quote }}
```

Comment:
I don't see anything that obsoletes these comments?

Reply:
I'd still like to see a comment that points at the real docs though.
(And I don't trust the _official_ docs, because that only exists for CAP, and not SCF)

Reply:
The replacement comment looks good enough (basically by existing).